.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .nyc_output
 coverage
-*.log
\ No newline at end of file
+*.log
+*lock*
```

Comment:
this is a bit too aggressive, please change it to `package-lock.json` explicitly

Reply:
Right